src/utils.js

```diff
@@ -126,14 +127,23 @@ export function getVariable(context, name) {
  *
  */
 export function gettext(str) {
-  return str;
+  const locale = process.env.LANG || process.env.LANGUAGE || process.env.LC_ALL || process.env.LC_MESSAGES || 'en-US';
+  let i18ndata = {};
+  try {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    i18ndata = require(`./locale/${locale}/messages.js`);
+  } catch (err) {
+    log.info('Initialize locales using extract-locales command');
+  }
+  const jed = new Jed(i18ndata);
+  return jed.gettext(str);
 }
 
 /*
  * An sprintf to use with gettext. Imported from Jed for when we have a proper
  * l10n solution.
  */
-export const { sprintf } = jed;
+export const { sprintf } = Jed;
```

Comment:
This probably won't work (needs a test, more on that later) since it doesn't re-use the initialized `jed` instance from inside `gettext` and will only work if you make the `const jed = Jed(i18ndata);` from earlier global and use `jed.sprintf` here, so this will become `export const sprintf = jed.sprintf;` or so (sorry, untested).